<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sm4 file based image</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54)</string>
  </property>
  <widget class="QLabel" name="cmap_label">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>15</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
background-color:rgb(100,100,100); font-size:14pt
</string>
   </property>
   <property name="text">
    <string>Choose the color map:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>350</y>
     <width>625</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255); font-size:28pt
</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="cmap_button1">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>80</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color:rgb(255,255,255);background-color:rgb(100,100,100); font-size:14pt</string>
   </property>
   <property name="text">
    <string>AFMhot</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">cmap_buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="cmap_button2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>110</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
background-color:rgb(100,100,100); font-size:14pt</string>
   </property>
   <property name="text">
    <string>hot</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">cmap_buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="cmap_img">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>70</y>
     <width>240</width>
     <height>95</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/cmaps.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/testing exes/cmaps.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="logo.qrc">:/newPrefix/cmaps.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="equa_label">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>180</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);background-color:rgb(100,100,100); font-size:14pt</string>
   </property>
   <property name="text">
    <string>Equalizing parameter: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="equa_write1">
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>235</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(100,100,100);color:rgb(230,230,230); font-size:14pt</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="equa_label1">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>235</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(100,100,100);color:rgb(255,255,255); font-size:14pt</string>
   </property>
   <property name="text">
    <string>Lower data threshold</string>
   </property>
  </widget>
  <widget class="QLabel" name="equa_label2">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>285</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(100,100,100);color:rgb(255,255,255); font-size:14pt</string>
   </property>
   <property name="text">
    <string>Upper data threshold</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="equa_write2">
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>285</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(100,100,100);
color:rgb(230,230,230); font-size:14pt</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="equa_perc2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>295</y>
     <width>20</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(100,100,100);color:rgb(255,255,255); font-size:14pt</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="equa_perc1">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>245</y>
     <width>20</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(100,100,100);
color:rgb(255,255,255); font-size:14pt</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="obj_type_label">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255); font-size:14pt
</string>
   </property>
   <property name="text">
    <string>SM4 object type:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="obj_type_micro_button">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>70</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
background-color:rgb(100,100,100); font-size:14pt</string>
   </property>
   <property name="text">
    <string>Microscopy</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="obj_type_spectro_button">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>110</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255); font-size:14pt</string>
   </property>
   <property name="text">
    <string>Spectroscopy</string>
   </property>
  </widget>
  <widget class="QLabel" name="dir_label">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>180</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255); font-size:14pt
</string>
   </property>
   <property name="text">
    <string>Directory:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="dir_button1">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>235</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255); font-size:14pt</string>
   </property>
   <property name="text">
    <string>Only current directory</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="dir_button2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>275</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255); font-size:14pt</string>
   </property>
   <property name="text">
    <string>Include inner directories</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>15</y>
     <width>385</width>
     <height>325</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(100,100,100)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>cmap_img</zorder>
  <zorder>cmap_label</zorder>
  <zorder>start_button</zorder>
  <zorder>cmap_button1</zorder>
  <zorder>cmap_button2</zorder>
  <zorder>equa_label</zorder>
  <zorder>equa_write1</zorder>
  <zorder>equa_label1</zorder>
  <zorder>equa_label2</zorder>
  <zorder>equa_write2</zorder>
  <zorder>equa_perc2</zorder>
  <zorder>equa_perc1</zorder>
  <zorder>obj_type_label</zorder>
  <zorder>obj_type_micro_button</zorder>
  <zorder>obj_type_spectro_button</zorder>
  <zorder>dir_label</zorder>
  <zorder>dir_button1</zorder>
  <zorder>dir_button2</zorder>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="cmap_buttonGroup"/>
 </buttongroups>
</ui>
